packages/theme/src/themes.ts

```diff
@@ -111,7 +112,7 @@ export const darkTheme = createTheme({
       info: '#2E77D0',
       error: '#E22134',
       text: '#FFFFFF',
-      link: '#000000',
+      link: '#FDFB9F',
```

Comment:
This color didn't seem to come from @katz95 's suggestion; is this a different one? Apols not sure where this one pops up, or why moving it away from black.

Reply:
Ah quite right, i reverted this back since I am honestly not sure what this will look like together with the banner and we should tackle that separately. Done!